### ISSUE TITLE
Content behind the active modal/popup remains focusable

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#329]
Content behind the active modal/popup shouldn't be focusable. When tabbing through the modal's controls, after the submit button the tabbing order returns to the top of the page rather than the modal's content.

People with mobility impairments who must rely on keyboard access for operating a page benefit from a logical, usable focus order. People with disabilities that make reading difficult can become disoriented when tabbing takes focus someplace unexpected. They benefit from a logical focus order. People with visual impairments can become disoriented when tabbing takes focus someplace unexpected or when they cannot easily find the content surrounding an interactive element.

**Comment by bmbc-dev-detec** [comment]
Ready for retesting after completion of #416

**Comment by bmbc-dev-detec** [comment]
Closing. Will add any problem found after retesting as new issues
